tools/python/m2k-osc/ain_tab.py

```diff
@@ -0,0 +1,199 @@
+import matplotlib
+import numpy as np
+import tkinter as tk
+from tkinter import ttk
+import matplotlib.figure as  fg
+import libm2k
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2Tk
+from matplotlib import style
+import threading
+import time
+import ctx_tab
+import aout_tab
+LARGE_FONT = ("Verdana",12)
+SMALL_FONT = ("Vedana",10)
+
+samples_queuech0=[]
+samples_queuech1=[]
+fig=fg.Figure(figsize=(8,6))
```

Comment:
it's a good idea to be consistent about style;
so, for example, when assigning stuff:   `SMALL_FONT__space__=__space__("Vedana",10)` vs `samples_queuech0__no_space__=__no_space__[]` 

not a big deal; but consistency looks nice;